### PR TITLE
Add patch for a segfault while outputting xml.

### PR DESCRIPTION
**Pull request by bluescarni**
See https://github.com/doxygen/doxygen/pull/555

We ran into this issue while building the docs for pagmo2 on travis, using the doxygen package from conda.

### CODE REVIEW

**Comment by isuruf** [comment]
I've created https://github.com/conda/conda-build/issues/1887 for the weird windows issue

**Review by isuruf — COMMENTED** [review]
Can you increase the build number?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -23,9 +23,12 @@ source:
     # CMake finds iconv_open in glibc, but finds iconv.h from libiconv.
     # Make sure both are found from one place.
     - find_iconv.patch                                                           # [not win]
+    # Doxygen 1.8.13 sometimes segfaults when building the XML output:
+    # https://github.com/doxygen/doxygen/pull/555
+    - doxygen-1.8.13-NULL-dereference.patch
```
Comment (isuruf) [inline recipe/meta.yaml]: Ah, found the error. Can you add a selector for this? `# [not win]`

Comment (tadeu) [reply recipe/meta.yaml]: Good catch! If someone gets this problem on Windows, this recipe will have to be changed to build on source for Windows too.

Comment (isuruf) [reply recipe/meta.yaml]: Yes, now that there is the clangdev package in conda-forge, that is possible.

**Pull request merged**